README.md

```diff
@@ -1,5 +1,5 @@
 <a href="https://travis-ci.org/catalyst/moodle-auth_saml2">
-<img src="https://travis-ci.org/catalyst/moodle-auth_saml2.svg?branch=master">
```

Comment:
I think this should be a github badge similar to changes made here?
https://github.com/catalyst/moodle-auth_saml2/pull/636/files

Reply:
We don't have the github workflow for 27_34 So I still use the travis-ci. one

Reply:
Ah okay. All good then.